decidim-core/app/commands/decidim/amendable/accept.rb

```diff
@@ -36,35 +37,15 @@ def call
 
       private
 
+      attr_reader :form
+
       def accept_amendment!
-        @amendment = Decidim.traceability.update!(
```

Comment:
Why are all the traceability mentions removed? I think this should come back

Reply:
Thanks. Will do.